maquetado/books-list.html

```diff
@@ -9,6 +9,10 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
+  <div class="books-list-container">
+    <div class="navbar">
+      <img src ="./assets/navbar.png" class="navbar-logo">
+    </div>
```

Comment:
I wouldn't set this inside the container. Generally the navbar it's in the outermost area. You can put in right after the body and you should use `<nav> ... </nav>` tag instead.